### DISCUSSION TITLE
Metron vs ComicVine data quality

### DISCUSSION THREAD

**Discussion by tehguitarist** [comictagger/comictagger#608] [General]
Hey, I did a google, and couldn't find anyone talking about this, but does anyone have any thoughts on the data quality from metron vs ComicVine? I've tested a couple of issues, and seem to get more genre tags, but it does seem to take a good deal longer to find the issues too.

Just curious as once StoryArc Numbers are implemented, I'll likely do a retag of my library

**Comment by bpepple** [comment]
> Hey, I did a google, and couldn't find anyone talking about this, but does anyone have any thoughts on the data quality from metron vs ComicVine? I've tested a couple of issues, and seem to get more genre tags, but it does seem to take a good deal longer to find the issues too.
> 
In general, Metron most likely has better information (things like genre, price, UPC, ISBN, reprint info, etc), but has **significantly** less issues indexed so everything in your collection might not be available on Metron.

Regarding, tagging issues the `metron_talker`plugin takes 2-4 times longer to identify issues compared to other tools (metron-rs, metron-tagger, etc), but is currently being worked on to fix these deficiencies.
